erpnext/www/lms/course.py

```diff
@@ -20,7 +20,18 @@ def get_context(context):
 	context.topics = course.get_topics()
 	context.has_access =  utils.allowed_program_access(context.program)
 	context.progress = get_topic_progress(context.topics, course, context.program)
+	context.total_progress = calculate_topic_progress(context.topics,context.progress)
+
 
 def get_topic_progress(topics, course, program):
 	progress = {topic.name: utils.get_topic_progress(topic, course.name, program) for topic in topics}
 	return progress
+
+def calculate_topic_progress(topics,topic_progress):
+	total_topics = len(topics)
+	completed_topic = 0
+	for topic in topic_progress:
+		if topic_progress[topic].get("completed"):
```

Comment:
```suggestion
		if topic.get("completed"):
```

Reply:
We can't use **topic.get("completed")** instead of **topic_progress[topic].get("completed"):**

If I made the changes which you suggested then it will throw an error and we can't achieve the expected result.
To get accurate output we have to use **if topic_progress[topic].get("completed"):**